tools/README.adoc

```diff
@@ -0,0 +1,121 @@
+[%hardbreaks]
+:toc: left
+:toclevels: 3
+
+
+== Overview
+
+This folder contains the following tools:
+
+. `sgcollect_info` -- a tool that collects diagnostics from Sync Gateway, such as logs, heap profile, cpu profile, expvars, etc.
+. A copy of `cbcollect_info` and `tasks.py`, which was refactored from the upstream `cbollect_info` in the nsserver repository, which has since diverged from this version.  `sgcollect_info` depends on the `cbcollect_info` in this repo.
+
+
+== SG Collectinfo
+
+=== Install dependencies (optional)
+
+To improve the quality of the collected zip files, install the following dependencies:
+
+- Go -- ideally use the version of Go that was used to build Sync Gateway
```

Comment:
Would be awesome if we could drop in a table here over SG version -> Go version

Reply:
Dropped a table in, but I can't figure out the SG 1.5 version of Go that was used to build it.

@ceejatec / @kimtrang do you know how to find out which version of Go was used to build SG 1.5?

Reply:
@tleyden you can use the delve debugger to evaluate `runtime.buildVersion`. Here's an example using SG 1.5.1 CE.

```
✔ ~/Downloads/couchbase-sync-gateway/bin
16:58 $ dlv exec sync_gateway
Type 'help' for list of commands.
(dlv) b main.main
Breakpoint 1 set at 0x20b3 for main.main() /Users/jenkins/jenkins/workspace/sgw-unix-build@2/1.5.1/community/godeps/src/github.com/couchbase/sync_gateway/main.go:28
(dlv) c
> main.main() /Users/jenkins/jenkins/workspace/sgw-unix-build@2/1.5.1/community/godeps/src/github.com/couchbase/sync_gateway/main.go:28 (hits goroutine(1):1 total:1) (PC: 0x20b3)
(dlv) p runtime.buildVersion
"go1.7.4"
```

Reply:
Neat!